testing/matrix/linter/rules/rules.go

```diff
@@ -324,6 +325,28 @@ func objectRecommendedLabels(object storage.StoreObject) errors.LintRuleError {
 	return errors.EmptyRuleError
 }
 
+func namespaceLabels(object storage.StoreObject) errors.LintRuleError {
+	if object.Unstructured.GetKind() != "Namespace" {
+		return errors.EmptyRuleError
+	}
+
+	if !strings.HasPrefix(object.Unstructured.GetName(), "d8-") {
+		return errors.EmptyRuleError
+	}
+
+	labels := object.Unstructured.GetLabels()
+
+	if _, ok := labels["prometheus.deckhouse.io/rules-watcher-enabled"]; ok {
```

Comment:
We need to add checking values. Value should be `true`.
https://github.com/deckhouse/deckhouse/pull/4407/files#diff-cb1d594f5b70f5cd683a082b4666e7246e15fe7a9638cc4d273aa7e5e7bd79c8R693

Reply:
done